src/logger/logger.js

```diff
@@ -73,37 +73,31 @@ export function enableDebugLogger() {
 }
 
 const captureBreadcrumb = (breadcrumb: any) => {
-  // FIXME
-  /*
   if (!process.env.STORYBOOK_ENV) {
     try {
-      if (typeof window !== 'undefined') {
-        require('sentry/browser').captureBreadcrumb(breadcrumb)
+      if (typeof window !== "undefined") {
+        require("~/sentry/browser").captureBreadcrumb(breadcrumb);
       } else {
-        require('sentry/node').captureBreadcrumb(breadcrumb)
+        require("~/sentry/node").captureBreadcrumb(breadcrumb);
```

Comment:
to be checked the implication of this in the runtime load (does it create an extra load time?). i guess it's fine, just to know if you checked a bit that.

Reply:
Yes it will as it's not bundled. should be pretty small thought. I implemented **raven** exactly as it use to be, didn't take any time to optimize. we should take some time to upgrade to **sentry** and to clean a bit the implementation :)